init.lua

```diff
@@ -345,6 +345,9 @@ function throwing.register_bow(name, def)
 			if shoot_arrow(itemstack, user, bow_index, def.throw_itself, new_stack) then
 				if not minetest.settings:get_bool("creative_mode") then
 					itemstack:add_wear(65535 / (def.uses or 50))
+					if use_toolranks then
+						toolranks.new_afteruse(itemstack, user, nil, {wear= 65535 / (def.uses or 50)})
```

Comment:
As this expression is used twice, it may make sense to use an auxiliary variable:
```lua
uses = def.uses or 50
```
so that the default value for `uses` is hard-coded only once.